add a LogDiscardedSpansConfig to the distributor to log discarded spa… (#3964)
grafana/tempo: docs/sources/tempo/troubleshooting/max-trace-limit-reached.md

The two most likely causes of refused spans are unhealthy ingesters or trace limits being exceeded.

To log spans that are discarded, add the `--distributor.log_discarded_spans.enabled` flag to the distributor or 
adjust the [distributor config]({{< relref "../configuration#distributor" >}}):

```yaml
distributor:
  log_discarded_spans:
    enabled: true
    include_all_attributes: false # set to true for more verbose logs
```

> This will log all discarded spans, as shown below:

Reviewer: Clarifying the wording. Is this correct? 
```suggestion
Adding the flag logs all discarded spans, as shown below:
```

```

In this case, use available configuration options to [increase limits]({{< relref "../configuration#ingestion-limits" >}}).

## Client Resets Connection

> When the client resets the connection before the distributor can consume the trace data, you will see logs like this:

Reviewer: ```suggestion
When the client resets the connection before the distributor can consume the trace data, you see logs like this:
```
